pkgs/applications/kde/marble.nix

```diff
@@ -18,7 +18,7 @@ mkDerivation {
     protobuf_21 qtscript qtsvg qtquickcontrols qtwebengine shared-mime-info krunner kparts
     knewstuff gpsd
   ];
-  preConfigure = ''
-    cmakeFlags+=" -DINCLUDE_INSTALL_DIR=''${!outputDev}/include"
```

Comment:
What is !outputDev? 😶‍🌫️

Reply:
`${!foo}` is the contents of the variable whose name is `$foo`. You know, [like PHP](https://www.php.net/manual/en/language.variables.variable.php).

Reply:
No